docs/source/analyze/distribution.md

```diff
@@ -42,6 +42,12 @@ spec:
           - pass:
               when: "= digitalocean"
               message: DigitalOcean is a supported distribution
+          - warn:
+              when: "= minikube"
+              message: Minikube is not suitable for production environments
+          - warn:
+              when: "= ibm"
+              message: IBM is not yet supported by kots
```

Comment:
These aren't "KOTS supported" but are an example of the distribution detection analyzer. This change seems too KOTS-specific for an example in the troubleshoot.sh repo.

Reply:
sure, makes sense